guide/payments/units-and-symbols.md

```diff
@@ -0,0 +1,147 @@
+    - file: home-unit-hidden
+      alt: Home screen showing user funds blurred out and unrecognizable.
+      caption: Hidden display blurs the balance to make the numbers unrecognizable.
+    - file: home-unit-options
+      alt: Home screen with a quick-access overlay to switch between units.
+      caption: Tapping and holding the balance gives quick access to unit options.
+    - file: enter-amount
+      alt: Screen for entering an amount of funds to send.
+      caption: When entering amounts, a dedicated UI element may be needed to switch between units.
+image: /assets/images/guide/payments/units-and-symbols/units-and-symbols-preview.jpg
+
+---
+
+<!--
+
+Editor's notes
+
+This page outlines basic considerations for displaying bitcoin and other currency amounts. Due to the huge
+amount of different notation standards around the world, it focuses on bitcoin.
+
+Illustration sources
+
+https://www.figma.com/file/VB3GQdAnhl8yta44DY3PSV/Bitcoin-Wallet-UI-Kit?node-id=2318%3A61126
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/payments/units-and-symbols/units-and-symbols.png"
+   retina = "/assets/images/guide/payments/units-and-symbols/units-and-symbols@2x.png"
+   mobile = "/assets/images/guide/payments/units-and-symbols/units-and-symbols-mobile.png"
+   mobileRetina = "/assets/images/guide/payments/units-and-symbols/units-and-symbols-mobile@2x.png"
+   alt-text = "Bitcoin value with annotations for parts of the number display."
+   caption = "Aspects to consider when formatting bitcoin amounts."
+   width = 1600
+   height = 700
+   layout = "full-width"
+%}
+
+# Units and symbols
+
+Bitcoin, bits, sats. The format and presentation of bitcoin values are probably amongst the most challenging for new users. Here we provide observations and recommendations on current usage in the context of established, global standards. The surest way to achieve wide-scale adoption is to formalize what is already being used today.
+
+Despite the challenge of the fractional aspect and the abundance of digits used to present the value of bitcoin, the intent is to serve as a guide to format the values so they can easily be scanned, quick to read, and intuitive to remember.
+
+#### Current adoption
+
+The most common way bitcoin is expressed is as BTC (bitcoin) or sat (satoshi), with 1 bitcoin being 100 000 000 satoshi. While not as common, denominations of BTC such as mBTC ("millibitcoins"), μBTC ("bits") are also sometimes used, as well as millisatoshi (msat) in the Lightning network. The unicode symbol ₿, formalized in June 2017, can also be used to represent BTC or bitcoin, but few typefaces support at the time of this writing. The chart below illustrates how each unit relates to the bitcoin unit.
+
+| Unit         | Symbol       | Bitcoin value     |
+| ------------ | ------------ | ----------------- |
+| bitcoin      | BTC or ₿     | 1.00 000 000      |
+| millibit     | mBTC         | 0.00 100 000      |
```

Comment:
```suggestion
| millibit     | mBTC         | 0.001             |
```

Reply:
I found it easier to parse when all 8 digits are visible on this table. You?

Reply:
Perhaps it's easier to parse with all 8 digits. Just thought people will take it as a guide to how it should appear in the UI (for which I think trailing zeros are never helpful)

Reply:
thinking about this a bit more, I think the trailing 0s are helpful, and standard in the current fiat system.

Reply:
It might also be a matter of context. If your use case requires parsing and comparing many numbers, you want a consistent format. You may also want it if your layout uses tables with right-aligned numbers, so they are easier to compare vertically (let's say a [block explorer](https://mempool.space/block/00000000000000000009f0e2bbc74a267e20eebc413239c30d12a780e7d21fd2)). Other places, like the donation boxes on [bitcoinsmiles.org](https://bitcoinsmiles.org), trimming feels right.

Seems like this whole page is basically saying "it depends" :)

Reply:
Yes, we are not providing many recommendations at all. I'm not precious about cutting the trailing zeros in this column but would never present them in a UI unless it's a VERY specific use case. I'll let @GBKS make the call here.